### ISSUE TITLE
More accessibility support

### ISSUE THREAD

**Issue by oyeanuj** [davidchin/react-input-range#118]
Hi @davidchin, I noticed that the Airbnb Rheostat component has more accessibility support that this library could utilize as well on the Handle. 

https://github.com/airbnb/rheostat/blob/ec25183d4475e72ea3f18e7b34c8627fbe2386c0/src/Slider.jsx#L716-L719

Thoughts?

**Comment by davidchin** [comment]
Yeah that's cool, thanks for bringing this to my attention. I'm happy to accept a PR that improves the accessibility of the component. 😄
